manifests/plugin/write_graphite/carbon.pp

```diff
@@ -1,19 +1,20 @@
 # a single graphite backend
 define collectd::plugin::write_graphite::carbon (
-  $ensure                    = 'present',
-  Stdlib::Host $graphitehost = 'localhost',
-  Stdlib::Port $graphiteport = 2003,
-  Boolean $storerates        = true,
-  $graphiteprefix            = 'collectd.',
-  $graphitepostfix           = undef,
-  $interval                  = undef,
-  $escapecharacter           = '_',
-  Boolean $alwaysappendds    = false,
-  $protocol                  = 'tcp',
-  Boolean $separateinstances = false,
-  Boolean $logsenderrors     = true,
-  Integer $reconnectinterval = 0,
-  Boolean $reversehost       = false,
+  $ensure                      = 'present',
+  Stdlib::Host $graphitehost   = 'localhost',
+  Stdlib::Port $graphiteport   = 2003,
+  Boolean $storerates          = true,
+  $graphiteprefix              = 'collectd.',
+  $graphitepostfix             = undef,
```

Comment:
Could you add something to the existing tests in spec/defines/collectd_plugin_write_graphite_spec.rb to exercise this new parameter.